src/config.ts

```diff
@@ -82,180 +84,35 @@ import { LoginOptions, DeepPartial } from './auth0-session';
  *
  * @category Server
  */
-export interface Config {
-  /**
-   * The secret(s) used to derive an encryption key for the user identity in a session cookie and
-   * to sign the transient cookies used by the login callback.
-   * Use a single string key or array of keys for an encrypted session cookie.
-   * Can use env key SECRET instead.
-   */
-  secret: string | Array<string>;
-
-  /**
-   * Object defining application session cookie attributes.
-   */
-  session: SessionConfig;
-
-  /**
-   * Boolean value to enable Auth0's logout feature.
-   */
-  auth0Logout: boolean;
-
-  /**
-   *  URL parameters used when redirecting users to the authorization server to log in.
-   *
-   *  If this property is not provided by your application, its default values will be:
-   *
-   * ```js
-   * {
-   *   response_type: 'code',
```

Comment:
We'll need to keep these docs for most of these properties because the defaults are different.

If you generate the docs now, it'll say the default `response_type` is `id_token` when it should be `code` https://github.com/auth0/nextjs-auth0/blob/beta/src/auth0-session/config.ts#L40

I also wanted to add the corresponding env vars to each config option, eg

```js
    /**	
   * To opt-out of sending the library and node version to your authorization server	
   * via the `Auth0-Client` header. Default is `true	
   * You can also use the `AUTH0_IDP_LOGOUT` env var
   */	
  enableTelemetry: boolean;
```

Also `baseURL` has some Vercel specific docs